### PR TITLE
Feature/wallet separation

### PR DESCRIPTION
**Pull request by argakiig**
Squashed data.ldb splitting
WIP needs test validation and migration path from old to new

### CODE REVIEW

File: rai/node/bootstrap.cpp [hunk 1]
```diff
@@ -903,16 +903,16 @@ void rai::bootstrap_attempt::process_fork (MDB_txn * transaction_a, std::shared_
...
+					    rai::transaction transaction (this_l->node->store.environment, nullptr, false);
+					    auto account (this_l->node->ledger.store.frontier_get (transaction, root));
+					    if (!account.is_zero ())
+					    {
+						    this_l->requeue_pull (rai::pull_info (account, root, root));
+					    }
+					    else if (this_l->node->ledger.store.account_exists (transaction, root))
+					    {
+						    this_l->requeue_pull (rai::pull_info (root, rai::block_hash (0), rai::block_hash (0)));
+					    }
```
Comment (PlasmaPower) [inline rai/node/bootstrap.cpp]: Whitespace changes

File: rai/node/utility.hpp [hunk 2]
```diff
@@ -133,9 +133,9 @@ class mdb_val
 };
 
 /**
- * RAII wrapper of MDB_txn where the constructor starts the transaction
- * and the destructor commits it.
- */
+	* RAII wrapper of MDB_txn where the constructor starts the transaction
+	* and the destructor commits it.
+	*/
```
Comment (PlasmaPower) [inline rai/node/utility.hpp]: Whitespace changes (and others in this file).

**Comment by PlasmaPower** [comment]
We should be careful of deadlocks and race conditions caused by transactions for different databases, but I think we're okay for now.

**Comment by argakiig** [comment]
closing in favor of wallet_separation branch

**Pull request closed without merging**